assets/questions/i18n.js

```diff
@@ -87,8 +92,8 @@ def createListOfUniqueStrings(atom, size):
   }, {
     instructions: [
       [
-        "It's always important to think about the edge cases for your code. ",
-        'In this case, short strings are not abbreviated by this strategy. ',
+        "Next, consider possible edge cases for your code. One potential ",
+        'edge case is usually short strings. Does your code handle them properly? ',
```

Comment:
Does your code handle them properly? --> Check that your code handles this case properly, and submit it when you're convinced it does.

(or something more specific)